stdlib/public/SDK/Foundation/UUID.swift

```diff
@@ -83,8 +99,8 @@ public struct UUID : ReferenceConvertible, Hashable, Equatable, CustomStringConv
     
     fileprivate var reference: NSUUID {
         var bytes = uuid
-        return withUnsafePointer(to: &bytes) {
-            return NSUUID(uuidBytes: unsafeBitCast($0, to: UnsafePointer<UInt8>.self))
+        return withUnsafeMutablePointer(to: &bytes, as: UInt8.self, capacity: MemoryLayout.size(ofValue: bytes)) {
+            return NSUUID(uuidBytes: $0)
         }
```

Comment:
Oh, that was one I'd written just for UUID.swift since it has so many of the two operations together (convert to pointer and then cast). It's here—could be a nice addition, but I don't know that it's necessary: https://github.com/apple/swift/pull/7329/files/cf260a26173985f867fb33cf52c808ec481aad44#diff-9e8b701cacc2a5ba71da8979a386a127R16